tox.ini

```diff
@@ -9,6 +9,8 @@ pip_pre = true
 deps =
     .[test]
 commands =
+    pytest --cov=zepben.eas --cov-report=xml --cov-branch
+    bash -ec 'if [[ -n $COVERALLS_REPO_TOKEN ]]; then coveralls; fi'
```

Comment:
we don't need coveralls anymore, we're using codecov, which you should be able to change to look like the [evolve-sdk-python tox.ini](https://github.com/zepben/evolve-sdk-python/blob/main/tox.ini)